lldb/tools/debugserver/source/MacOSX/MachThread.h

```diff
@@ -126,6 +127,8 @@ class MachThread {
 protected:
   static bool GetBasicInfo(thread_t threadID,
                            struct thread_basic_info *basic_info);
+  static bool GetExtendedInfo(thread_t threadID,
+                              struct thread_extended_info *extended_info);
 
   bool GetIdentifierInfo();
```

Comment:
`MachThread::GetIdentifierInfo` is now gone, right?

Reply:
Thanks, I didn't notice that.